Applications/Spire/Source/Ui/OverlayPanel.cpp

```diff
@@ -130,6 +158,17 @@ void OverlayPanel::keyPressEvent(QKeyEvent* event) {
   QWidget::keyPressEvent(event);
 }
 
+bool OverlayPanel::is_ancestor(QWidget* widget) {
```

Comment:
https://doc.qt.io/qt-5/qwidget.html#isAncestorOf

Reply:
I tried that originally but it only checks up until the window containing the given widget, while my implementation of it goes  up to the the highest parent regardless of what window it's in.